st2api/st2api/controllers/v1/webhooks.py

```diff
@@ -119,70 +106,41 @@ def get_one(self, name):
         # For demonstration purpose return 1st
         return triggers[0]
 
-    @request_user_has_webhook_permission(permission_type=PermissionType.WEBHOOK_SEND)
-    @jsexpose(arg_types=[str], status_code=http_client.ACCEPTED)
-    def post(self, *args, **kwargs):
-        hook = '/'.join(args)  # TODO: There must be a better way to do this.
-
-        # Note: For backward compatibility reasons we default to application/json if content
-        # type is not explicitly provided
-        content_type = pecan.request.headers.get('Content-Type', 'application/json')
-        content_type = parse_content_type_header(content_type=content_type)[0]
-        body = pecan.request.body
-
-        try:
-            body = self._parse_request_body(content_type=content_type, body=body)
-        except Exception as e:
-            self._log_request('Failed to parse request body: %s.' % (str(e)), pecan.request)
-            msg = 'Failed to parse request body "%s": %s' % (body, str(e))
-            return pecan.abort(http_client.BAD_REQUEST, msg)
-
-        headers = self._get_headers_as_dict(pecan.request.headers)
+    # @request_user_has_webhook_permission(permission_type=PermissionType.WEBHOOK_SEND)
```

Comment:
It looks like we are not performing RBAC check here anymore? Or am I missing something?